images.yaml

```diff
@@ -1,9 +1,10 @@
 - name: alpine
   patterns:
-  - pattern: '>= 3.9'
+  - pattern: '>= 3.11'
     customImages:
     - tagSuffix: giantswarm
       dockerfileOptions:
+      - RUN apk add --no-cache ca-certificates
```

Comment:
@stone-z How does this work currently? Will this cause retagging of already existing tags?

Reply:
No - this won't be applied until a new tag matches the pattern